Packages/install_octoprint

```diff
@@ -67,4 +62,4 @@ WantedBy=multi-user.target
 EOF
 
 systemctl enable octoprint
-systemctl start octoprint
+#systemctl start octoprint
```

Comment:
woops, that was sucking major resources.
Why keep the comment though, and not remove it altogether?

Reply:
Just following the previous way it was done in the other modules.
I'd be happy to eliminate it if no one needs it as a "hint"